src/main/java/microsoft/exchange/webservices/data/AutodiscoverEndpoints.java

```diff
@@ -15,47 +15,11 @@
  */
 enum AutodiscoverEndpoints {
 
-  /**
-   * No endpoints available.
-   */
-  None(0),
+  None,
+  Legacy,
+  Soap,
+  WsSecurity,
+  WSSecuritySymmetricKey,
+  WSSecurityX509Cert;
```

Comment:
why did you remove the javadoc here? They where not very declarative but maybe just the content would need to be changed.